components/esp_eth/include/esp_eth_phy.h

```diff
@@ -278,38 +278,47 @@ esp_eth_phy_t *esp_eth_phy_new_dp83848(const eth_phy_config_t *config);
 /**
 * @brief Create a PHY instance of KSZ8041
 *
+* @warning: This function is deprecated and is kept here only for compatibility reasons.
+*           Use esp_eth_phy_new_ksz80xx instead.
+*
 * @param[in] config: configuration of PHY
 *
 * @return
 *      - instance: create PHY instance successfully
 *      - NULL: create PHY instance failed because some error occurred
 */
-esp_eth_phy_t *esp_eth_phy_new_ksz8041(const eth_phy_config_t *config);
+esp_eth_phy_t *esp_eth_phy_new_ksz8041(const eth_phy_config_t *config)  __attribute__ ((deprecated));
```

Comment:
Since we are now about to release the IDV v5.0, we are allowed to make breaking changes. You can remove these functions.

Reply:
Ok, I will do that. Unfortunately, I don't know where breaking changes (and all the other release notes mentioned on the "GitHub Releases" page) are documented.

Reply:
No worry about release notes and breaking change description. I'll take care of it.

Reply:
Ok, thanks. But for interest: Is it maintained in the repo files itself?

Reply:
Migration guide is located [here](https://github.com/espressif/esp-idf/tree/master/docs/en/migration-guides).
Release notes are parsed from description of Merge Requests in our internal Gitlab.